.github/workflows/skip.yml

```diff
@@ -29,9 +29,15 @@ jobs:
     name: ${{ matrix.os }} / ${{ matrix.python-version }}
```

Comment:
We shouldn't need changes here as we use this to trick the mandatory action checks. And this won't be mandatory until 3.10 is stable.